Sources/active-win/main.swift

```diff
@@ -1,6 +1,33 @@
 import AppKit
 
-// This shows the system prompt if there's no permission.
+// Uses AppleScript to get the browser URL from Chrome or Safari
+func getBrowserURL(_ appName: String) -> String? {
+	// Gets the appropriate AppleScript for each browser type
+	guard let scriptText = getScriptText(appName) else { return nil }
+	
+	// Prepare and execute AppleScript script to query URL
+	var error: NSDictionary?
+	guard let script = NSAppleScript(source: scriptText) else { return nil }
+	guard let outputString = script.executeAndReturnError(&error).stringValue else { return nil }
+	
+	return outputString
+}
+
+// Formats the AppleScript text for Chrome and Safari
+func getScriptText(_ appName: String) -> String? {
```

Comment:
The method name needs to be more specific.

Reply:
Updated to `getActiveBrowserTabURLAppleScriptCommand`